src/Command/Import/ImportCommand.php

```diff
@@ -108,7 +110,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'delicious' => $this->deliciousImport,
             'elcurator' => $this->elcuratorImport,
             'shaarli' => $this->shaarliImport,
-            'pocket' => $this->pocketHtmlImport,
+            'pocket_html' => $this->pocketHtmlImport,
```

Comment:
We should keep the default value to avoid any confusion for people actually using it.
```suggestion
            'pocket' => $this->pocketHtmlImport,
```